### PR TITLE
IR-2985-Point-Click-doesnt-work-with-Assets-Panel

### PR DESCRIPTION
**Pull request by MichaelEstes**
## References
closes #_insert number here_

### CODE REVIEW

File: packages/ui/src/components/editor/panels/Assets/container/index.tsx [hunk 1]
```diff
@@ -70,7 +79,13 @@ type Category = {
   depth: number
 }
 
-const AssetsPreviewContext = createContext({ onAssetSelectionChanged: (props: AssetSelectionChangePropsType) => {} })
+const AssetsPreviewContext = hookstate({
```
Comment (HexaField) [inline packages/ui/src/components/editor/panels/Assets/container/index.tsx]: any reason we arent using hyperflux here? or making it a scoped useHookstate?

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/editor/panels/Assets/container/index.tsx (hunk 1 above)
Comment (MichaelEstes) [reply packages/ui/src/components/editor/panels/Assets/container/index.tsx]: Nope, switched it to a hookstate

**Review by HexaField — CHANGES_REQUESTED** [review]
This doesn't seem to be working for me. I opened the asset panel, switch to Click & Place mode, but i was not able to select an asset and place, as if it was still in drag and drop mode. When i switched to the file view, i was able to click and place, and when i then switched to assets the old file selected from the file view was still click and placing.

**Comment by HexaField** [comment]
> This doesn't seem to be working for me. I opened the asset panel, switch to Click & Place mode, but i was not able to select an asset and place, as if it was still in drag and drop mode. When i switched to the file view, i was able to click and place, and when i then switched to assets the old file selected from the file view was still click and placing.

Oh! Actually, it just looks like `.glb` files do not trigger the click and place, i was able to select .fbx and .gltf files and place those from the asset panel

**Comment by HexaField** [comment]
> > This doesn't seem to be working for me. I opened the asset panel, switch to Click & Place mode, but i was not able to select an asset and place, as if it was still in drag and drop mode. When i switched to the file view, i was able to click and place, and when i then switched to assets the old file selected from the file view was still click and placing.
> 
> Oh! Actually, it just looks like `.glb` files do not trigger the click and place, i was able to select .fbx and .gltf files and place those from the asset panel

Actually, on further inspection, it just seems like certain files are not changing the active asset to click and place. Everything else I said seems downstream from that.

**Comment by MichaelEstes** [comment]
> > > This doesn't seem to be working for me. I opened the asset panel, switch to Click & Place mode, but i was not able to select an asset and place, as if it was still in drag and drop mode. When i switched to the file view, i was able to click and place, and when i then switched to assets the old file selected from the file view was still click and placing.
> > 
> > 
> > Oh! Actually, it just looks like `.glb` files do not trigger the click and place, i was able to select .fbx and .gltf files and place those from the asset panel
> 
> Actually, on further inspection, it just seems like certain files are not changing the active asset to click and place. Everything else I said seems downstream from that.

Not all assets are valid for click placement, I've added some warning messaging around that. GLBs not working was a bug though, those should work now. I have no idea how you got an FBX file to work.

**Review by HexaField — APPROVED** [review]

**Pull request merged**